lib/worker.js

```diff
@@ -58,14 +60,8 @@ function shim(Linter) {
   return LinterShim;
 }
 
-function loadDefaultTSLint() {
-  if (!tslintDef) {
-    // eslint-disable-next-line import/no-dynamic-require
-    tslintDef = require(tslintModuleName).Linter;
-  }
-}
-
-async function getLocalLinter(basedir) {
+async function resolveAndCacheLinter(fileDir, moduleDir) {
```

Comment:
This doesn't need to be marked as `async` if no `await`s are done within it

Reply:
I think this is my last comment =]. @Arcanemagus, any objections after this?

Reply:
Ah, true, since all it's doing is immediately returning a Promise, it doesn't really need to be marked as async. I'll go ahead and clean up those unnecessary `async` declarations on the functions that do that.

Reply:
Just as a note, I personally leave the `async` on there just to make it obvious to myself that the function needs to be `await`ed on where it gets called, not a huge issue though as you are perfectly
 right that it isn't needed 😉.